### PR TITLE
PLANNER-2472 daily native build of optaplanner-quickstarts (#256)

### PR DESCRIPTION
**Pull request by Ginxo**
cherry picked from https://github.com/kiegroup/optaplanner-quickstarts/commit/edc101fc0eded2b2d24f963a48ee4b8a7cf2fdf8

### JIRA

https://issues.redhat.com/browse/PLANNER-2472

### Referenced pull requests

* https://github.com/kiegroup/optaplanner-quickstarts/pull/256

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* for a <b>pull request build</b> please add comment: <b>Jenkins retest this</b>
* for a <b>full downstream build</b> please add the label `run_fdb`
* for a <b>specific pull request build</b> please add comment: <b>Jenkins (re)run [optaplanner-quickstarts] tests</b>
* for a <b>Quarkus LTS check</b> please add comment: <b>Jenkins run LTS</b>
* for a <b>specific Quarkus LTS check</b> please add comment: <b>Jenkins (re)run [optaplanner-quickstarts] LTS</b>
* for a <b>Native check</b> please add comment: <b>Jenkins run native</b>
* for a <b>specific Native LTS check</b> please add comment: <b>Jenkins (re)run [optaplanner-quickstarts] native</b>
</details>

### CODE REVIEW

File: .ci/jenkins/dsl/jobs.groovy [hunk 1]
```diff
@@ -54,3 +69,35 @@ void setupMultijobPrNativeChecks() {
 void setupMultijobPrLTSChecks() {
     KogitoJobTemplate.createMultijobLTSPRJobs(this, getMultijobPRConfig()) { return getDefaultJobParams() }
 }
+
+void setupNativeJob() {
```
Comment (radtriste) [inline .ci/jenkins/dsl/jobs.groovy:73]: you never call those methods in the DSL. You should add them.
See https://github.com/kiegroup/optaplanner/blob/main/.ci/jenkins/dsl/jobs.groovy#L80 and https://github.com/kiegroup/optaplanner/blob/main/.ci/jenkins/dsl/jobs.groovy#L94

**Comment by triceo** [comment]
@Ginxo Want to resolve Tristan's comment? Let's merge it after!

**Comment by rsynek** [comment]
@Ginxo do we still want to merge this PR or should it be closed?

**Comment by radtriste** [comment]
for now the optaplanner-quickstarts native is done as part optaplanner native nightly. So I doubt it is needed for now.
Also there is an ongoing work to reuse the build-chain in nightlies.

**Comment by rsynek** [comment]
Closing based on @radtriste 's comment. Feel free to reopen the PR if needed.

**Pull request closed without merging**